src/request/src/response.js

```diff
@@ -158,9 +158,9 @@ export class KinveyResponse extends Response {
       return super.error;
     }
 
-    if (isDefined(name)) {
-      error.name = name;
-    }
+    // if (isDefined(name)) {
```

Comment:
It is deliberate. I wanted to make sure it was ok before I removed the code. Maybe we should discuss in standup.